docs/components/board/pi5.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Configure a Raspberry Pi 5 board"
+linkTitle: "pi5"
+weight: 20
+type: "docs"
+description: "Configure a Raspberry Pi 5 board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Bucket, Alan
+---
+
+{{% alert title="REQUIREMENTS" color="note" %}}
+
+Follow the [setup guide](/get-started/installation/prepare/rpi-setup/) to prepare your Pi for running `viam-server` before configuring a `pi5` board.
+
+{{% /alert %}}
+
+Configure a `pi5` board to integrate a [Raspberry Pi 5](https://www.raspberrypi.com/products/raspberry-pi-5/) into your robot.
+
+To configure a Raspberry Pi 4 or earlier, see [Configure a Raspberry Pi 4, 3, or Zero 2 board](/components/board/pi/).
+
+### Enable hardware PWM
+
+_(Optional)_ If you want to use hardware PWM on {{< glossary_tooltip term_id="pin-number" text="pins" >}} 12 and 35, edit <file>/boot/config.txt</file> on your Pi, adding the following line:
```

Comment:
Not sure if this is worth pointing out, but for the moment we've set those pins to be PWM or nothing. If you don't do this, those pins will be totally useless. 

Olivia and (to a lesser extent) I are working to improve this, but it won't be done for a while. :confused:

Reply:
Okay I will add a sentence to that effect!